MQTTstuff.ino

```diff
@@ -17,20 +17,23 @@
   
 #ifdef USE_MQTT
   #include <PubSubClient.h>           // MQTT client publish and subscribe functionality
-  #define MQTT_WAITFORCONNECT 600000  // 10 minutes
-  #define MQTT_WAITFORRETRY     3     // 3 seconden backoff
+//  #define MQTT_WAITFORCONNECT 600000  // 10 minutes
+//  #define MQTT_WAITFORRETRY     3     // 3 seconden backoff
+
+  DECLARE_TIMER_MS(mqttTimer, 500);                       //every 500 ms do handle state
+  DECLARE_TIMER_SEC(mqttRetryTimer, 3);                   //backoff timer 
+  DECLARE_TIMER_MIN(timeMQTTReconnect, 10)                //try reconnecting cyclus timer
+  DECLARE_TIMER_SEC(timeMQTTPublish,  ((settingMQTTinterval == settingInterval) ? (settingMQTTinterval-1):settingMQTTinterval)); //special case, if telegram interval = mqtt interval, then mqtt interval needs to be shorter
```

Comment:
Waarom moet de MQTT interval altijd kleiner zijn dan de telegram interval?
Het heeft geen zin om een MQTT boodschap te versturen als er geen nieuw telegram is..
.. en nogmaals: het is géén realtime os. Alles wordt sequentieel/achter/na elkaar afgehandeld. 
MQTT interval mag gróter zijn dan telegram interval, maar nooit kleiner!